lib/hooks.js

```diff
@@ -99,8 +99,13 @@ function hookRunnerApplication (hookName, boot, server, cb) {
       if (i === 0 && c === 0) { // speed up start
         exit()
       } else {
+        // This is the last function executed for every fastify instance
         boot(function manageTimeout (err, done) {
+          // this callback is needed by fastify to provide an hook interface without error
+          // and managing the error for the user
```

Comment:
This is a little unclear to me. What does "provide an hook interface without error" mean?

Reply:
I mean without exposing the error argument as first parameter

I will try to reformulate

Reply:
Done